### DISCUSSION TITLE
Migrate from yarn to pnpm

### DISCUSSION THREAD

**Discussion by ffxsam** [katspaugh/wavesurfer.js#3412] [Ideas]
Anyone have any objections to migrating to pnpm? I could list the reasons, but a quick Google search for "yarn vs pnpm" would be better. 🙂 We've got a few different repos for our SaaS that used to use yarn, and since moving to pnpm, package updates & installation is way faster.

**Comment by katspaugh** [comment]
I don’t mind!

**Reply by ffxsam** [reply]
Excellent! I'll add it to my list.
